coala_quickstart/Strings.py

```diff
@@ -50,3 +50,15 @@
 
 You can see all of them here: {}
 """.format(BEAR_DOCS_URL)
+
+PRINT_BEARS = {'unusable':
+               {'msg': """
+\nBased on the configuration options the following
+bears have been identified to be unusable:
```

Comment:
an extra message on how to make these bears usable too would be good over here

Reply:
Ok, I have to change this, it sound better with the indications. Also, do you feel that using `PRINT_BEARS` as in the PR is an ok option? I feel that it somehow looks ugly (hacky), but on the other hand I feel that the other options either add duplicate code, or are less flexible?